core/src/main/se/tevej/game/model/systems/PopulationSystem.java

```diff
@@ -0,0 +1,104 @@
+package main.se.tevej.game.model.systems;
+
+import com.badlogic.ashley.core.Engine;
+import com.badlogic.ashley.core.Entity;
+import com.badlogic.ashley.core.Family;
+import com.badlogic.ashley.utils.ImmutableArray;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import main.se.tevej.game.model.components.InventoryComponent;
+import main.se.tevej.game.model.components.buildings.HomeComponent;
+import main.se.tevej.game.model.resources.NotEnoughResourcesException;
+import main.se.tevej.game.model.resources.Resource;
+import main.se.tevej.game.model.resources.ResourceType;
+
+public class PopulationSystem extends TSystem {
+
+    private Engine engine;
+    private float gestationProgress;
+    private int alivePop;
+    @SuppressFBWarnings(
+        value = "SS_SHOULD_BE_STATIC",
+        justification = "No need to be static and checkbugs will complain if it is."
+    )
+    private final int popHunger = 10;
+    @SuppressFBWarnings(
+        value = "SS_SHOULD_BE_STATIC",
+        justification = "No need to be static and checkbugs will complain if it is."
+    )
+    private final float gestationPeriod = 10f;
```

Comment:
What. This name.

Reply:
it is very self explanatory, you could always try to find a better name for a variable which decides how often children are made in homes

Reply:
Very descriptive.

Reply:
right. didn't know gestation was a word.